### DISCUSSION TITLE
插件安装后无法正常使用

### DISCUSSION THREAD

**Discussion by w125939224** [nonebot/discussions#176] [问答] [answered]
插件安装成功后无法使用。
nonebot使用脚手架安装，启动截图如下：
![1657760464136](https://user-images.githubusercontent.com/68673870/178862457-d2429184-742e-45d5-80dc-855323d53b6a.jpg)
![1657760505053](https://user-images.githubusercontent.com/68673870/178862467-62fe43d0-44a9-4379-a503-9610e3ed6bdd.jpg)
输入互动插件指令，截图如下：
![1657760733416](https://user-images.githubusercontent.com/68673870/178863413-f3694841-efc0-4100-9ecb-992f24957337.jpg)
![1657760769081](https://user-images.githubusercontent.com/68673870/178863438-18c30542-460c-4d54-a33a-8c637b025b71.jpg)

**Comment by yanyongyu** [comment, marked as answer]
status插件需要配置superusers，只有超管可以使用。根据你的配置项，其他命令应该以`/`开头，如`/命令名`

**Reply by w125939224** [reply]
谢谢大佬，完美解决~
